src/core/gltf2importer/gltf2context.cpp

```diff
@@ -370,6 +370,16 @@ void GLTF2Context::addLocalFile(const QString &file)
     m_localFiles.push_back(file);
 }
 
+QByteArray GLTF2Context::bufferChunk() const
+{
+    return m_bufferChunk;
+}
+
+void GLTF2Context::setBufferChunk(QByteArray &bufferChunk)
```

Comment:
const QByteArray & ? Since you are storing that in a plain QByteArray

Reply:
done